python/py_opt_control/min_time_bvp.py

```diff
@@ -226,3 +219,35 @@ def sample_min_time_bvp(p0, v0, a0, t, j, dt):
         st, sj, sa, sv, sp = t, j, a, v, p
 
     return st, sj, sa, sv, sp
+
+def uniformly_sample(p0, v0, a0, t, j, dt):
+    """
+    Given an initial state and an input switching sequence, compute the full
+    state at sampled times with resolution dt. 
+
+    Inputs:
+        p0, initial position,     shape=(N,)
+        v0, initial velocity,     shape=(N,)
+        a0, initial acceleration, shape=(N,)
+        t,  switch times, shape=(N,M)
+        j,  jerk,         shape=(N,M)
+        dt, time resolution
+    Outputs:
+        st, times,        shape=(N,K)
+        sj, jerk,         shape=(N,K)
+        sa, acceleration, shape=(N,K)
+        sv, velocity,     shape=(N,K)
+        sp, position,     shape=(N,K)
+    """
+
+    # It is expected that the final t for each axis are identical.
+    end_t = t[:,-1].max()
+    t[:,-1] = end_t
+    
+    # Allocate dense samples over time, jerk, acceleration, velocity, position.
+    st = np.arange(t[0,0], end_t, dt)
+    if st[-1] != end_t: # The final sample time gets exactly to the end state.
+        st = np.append(st, end_t)
+
+    _, sj, sa, sv, sp = sample(p0, v0, a0, t, j, st)
```

Comment:
The '_' output argument of sample() should be removed once the st output argument is remove from the sample() function.